ye.py

```diff
@@ -0,0 +1,27 @@
+from flask import Flask, render_template

```

Comment:
Why you created this file, initiating second Flask app?
And what this filename means?